resources/romfs/lang/it-IT.json

```diff
@@ -0,0 +1,291 @@
+{
+    "code": "it-IT",
+    "title": "Italiano",
+    "fallback": true,
```

Comment:
This should be false, we've already fallback to English.

Reply:
@tsl0922 

Sorry. I fixed it. I hope that now is OK.
Is it ok "it-IT" or should be only "IT"?
Thanks.

Reply:
It should be a [IETF language tag](https://en.wikipedia.org/wiki/IETF_language_tag), `it-IT` is ok.